configs/openmrs_config/concepts/general.csv

```diff
@@ -4,9 +4,11 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:k
 7c53975f-be65-499d-b82c-abe01bcd6eca,,,Referral Hospital,មន្ទីរពេទ្យបង្អែក,RH,មន្ទីរពេទ្យបង្អែក,,,Misc,N/A,,,,
 fda15e24-6474-4797-8b59-ce11d0df0bb9,,,Non Government Organization,អង្គការក្រៅរដ្ឋាភិបាល,NGO,អង្គការក្រៅរដ្ឋាភិបាល,,,Misc,N/A,,,,
 459b0fe3-62a1-42e3-bc9f-d22f1b1506ce,,,Self presented,មកដោយខ្លួនឯង,Self presented,មកដោយខ្លួនឯង,,,Misc,N/A,,,,
+6ee8979c-e559-443f-bf66-ee49cde114bd,,,VHSG (village health support group),ក្រុមទ្រទ្រង់សុខភាពភូមិ,VHSG (village health support group),ក្រុមទ្រទ្រង់សុខភាពភូមិ,,,Misc,N/A,,,,
```

Comment:
@kazlaw, what do you think if just only use the  full name `Village health support group`?

Reply:
Hi @kdaud, according to [this ](https://docs.google.com/spreadsheets/d/13PHkzorfTmYRjuIJv3fCml3zBAuDba6Mgdpu4H7cAg8/edit?pli=1#gid=0&range=50:50)it was provided in full